### PR TITLE
feat(github-growth): exclude hidden repos

### PR DESCRIPTION
**Pull request by cathteng**
Don't return any hidden repositories in GET OrganizationRepositories.

For ER-1719

### CODE REVIEW

**Review by schew2381 — APPROVED** [review]

File: tests/sentry/api/endpoints/test_organization_repositories.py [hunk 1]
```diff
@@ -48,6 +48,32 @@ def test_get_integration_repository(self):
...
+            provider="dummy",
+            config={"name": "getsentry/example"},
+        )
+        Repository.objects.create(
+            name="getsentry/sentry",
+            organization_id=self.org.id,
+            external_id=54321,
+            provider="dummy",
+            config={"name": "getsentry/sentry"},
+            status=ObjectStatus.HIDDEN,
```
Comment (nhsiehgit) [inline tests/sentry/api/endpoints/test_organization_repositories.py:65]: so `status` is an optional attr?

Is there already a test for multiple active repos?

**Review by nhsiehgit — APPROVED** [review]
lgtm

nonblocking question

File: tests/sentry/api/endpoints/test_organization_repositories.py (hunk 1 above)
Comment (cathteng) [reply tests/sentry/api/endpoints/test_organization_repositories.py:65]: no but i can add one

Comment (cathteng) [reply tests/sentry/api/endpoints/test_organization_repositories.py:65]: the default for status is `ObjectStatus.ACTIVE`

**Pull request merged**